<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LIN-114PBK3" failures="1" tests="2" name="Error Validation Test" time="12.301" errors="0" timestamp="2024-03-21T19:47:11 IST">
  <testcase classname="Selenium.ErrorValidationsTest" name="failedTest" time="0.57">
    <skipped/>
  </testcase> <!-- failedTest -->
  <testcase classname="Selenium.ErrorValidationsTest" name="failedTest" time="0.555">
    <failure type="java.lang.AssertionError" message="expected [Incorrect] but found []">
      <![CDATA[java.lang.AssertionError: expected [Incorrect] but found []
at Selenium.ErrorValidationsTest.failedTest(ErrorValidationsTest.java:11)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- failedTest -->
</testsuite> <!-- Error Validation Test -->
